gap/PolygonalComplexes/properties.gd

```diff
@@ -763,3 +763,25 @@ DeclareOperation( "IsChaoticVertex", [IsTwistedPolygonalComplex, IsPosInt] );
 DeclareOperation( "IsChaoticVertexNC", [IsTwistedPolygonalComplex, IsPosInt] );
 #! @EndGroup
 
+#! @BeginGroup FaceTwoColouring
+#! @Description
+#! The method returns a 2-face-colouring of a given polygonal complex if it exists
+#! or an error if it doesn't.
```

Comment:
Apparently, GitHub did not show my full feedback here. It would be nice if "fail" was written as `<K>fail</K>` in the documentation, since this ensures a slightly different formatting.